runtime/near-wallet-contract/implementation/wallet-contract/src/tests/sanity.rs

```diff
@@ -60,7 +60,7 @@ async fn test_insufficient_gas() -> anyhow::Result<()> {
             "target": target,
             "tx_bytes_b64": codec::encode_b64(&codec::rlp_encode(&signed_transaction))
         }))
-        .gas(near_gas::NearGas::from_tgas(7))
+        .gas(near_primitives::Gas::from_tera(7))
```

Comment:
This is definitely not going to work. This method is part of near-workspaces-rs, which definitely uses near-gas crate.